comps/lvms_pg/requirements.txt

```diff
@@ -0,0 +1,13 @@
+aiohttp
+docarray
+fastapi
+httpx
+opentelemetry-api
+opentelemetry-exporter-otlp
+opentelemetry-sdk
+prometheus-fastapi-instrumentator
+pyyaml
```

Comment:
Do the other components pin all the versions of their dependencies? Should we?

Reply:
No, they do not